internal/util/filesystem.go

```diff
@@ -75,6 +76,9 @@ func (u *FileSystemUtil) GetHomeDir() string {
 }
 
 func (u *FileSystemUtil) GetConfigDir() string {
+	if u.ConfigDir != "" {
```

Comment:
I want this to be configurable -- mostly for tests right now, but I could see this being configurable by the end user in the future.